src/commands/lint.ts

```diff
@@ -24,6 +24,14 @@ export class LintCommand implements Command {
 
   args = [
     {
+      name: 'input',
```

Comment:
Not sure I fully understand what this does but the issue I've been bumping into is wanting to lint my entire project but exclude certain directories (bower_components, for example)